authors/apps/articles/views.py

```diff
@@ -2,17 +2,23 @@
 from rest_framework.permissions import IsAuthenticatedOrReadOnly
 from rest_framework.response import Response
 from rest_framework import status
+from rest_framework.pagination import PageNumberPagination
 
 from .permissions import IsOwnerOrReadonly
 from .models import ArticlesModel, Comment
 from .serializers import ArticlesSerializers, CommentsSerializers
 from .renderers import ArticlesRenderer
 
+class StandardPagination(PageNumberPagination):
```

Comment:
@caveinn, nice work man. Just an observation though, kindly add `page_size` and `max_page_size` as environment variables so that we can accommodate easy changes to the same in the future. Thanks.